# file: C:\Users\matth\AppData\Roaming\Python\Python313\site-packages\numpy\_core\_exceptions.py
# hypothesis_version: 6.131.22

[1000, 1024, 'EiB', 'GiB', 'KiB', 'MiB', 'PiB', 'TiB', 'bytes', '{:#.0f} {}', '{:#.3g} {}', '{:.0f} {}', '{} ']